src/pages/wallet/index.tsx

```diff
@@ -0,0 +1,88 @@
+"use client";
+import { useState } from "react";
+import Navbar from "../../components/Navbar";
+import Footer from "../../components/Footer";
+import { Input } from "antd";
+import Button from "../../components/commons/Button";
+import { WarningOutlined } from "@ant-design/icons";
+
+function Wallet() {
+  const [publicKey, setPublicKey] = useState<string>("");
+  const [balance, setBalance] = useState<string>("0");
+
+  return (
+    <>
+      <div className="wallet-container flex flex-col min-h-screen">
+        <Navbar />
+        <div className="flex flex-col justify-start items-center gap-5 flex-grow py-3">
+          <div className="flex justify-evenly w-full gap-5 py-5 px-10">
+            <div className="flex flex-col items-center justify-center ">
+              <p className="wallet-balance-title text-sm sm:text-2xl">
+                Your balance
+              </p>
+              <p className="wallet-balance-amount text-sm sm:text-2xl font-bold">
+                {balance} Lumens (XML)
+              </p>
+            </div>
+            <div className="flex items-center justify-center gap-2 sm:w-[30%] w-[40%]">
+              <Button onClick={() => {}} buttonClass="button-send" fullWidth>
+                Send
+              </Button>
+              <Button
+                onClick={() => {}}
+                buttonClass="button-receive"
+                fullWidth
+                secondary
+              >
+                Receive
+              </Button>
+            </div>
+          </div>
+          <div className="bg-neutralMain flex flex-col px-[12.5%] text-black bg-blue-200 rounded-md border-2 border-blue-300 sm:h-[70vh] h-[70vh] sm:w-[60%]  hover:bg-blue-100">
+            <div className="flex items-center justify-center font-semibold gap-x-3">
+              <div className="w-[80%]">
+                <div className="wallet-publicKey-title flex items-start justify-start text-xs sm:text-xl font-semibold py-5">
```

Comment:
```suggestion
                <div className="wallet-public-key-title flex items-start justify-start text-xs sm:text-xl font-semibold py-5">
```

Reply:
solved in commit 511e7114d29729205dbe09561458400491ef7083.